src/libraries/System.Memory.Data/src/System/BinaryData.cs

```diff
@@ -22,6 +22,11 @@ public class BinaryData
         /// </summary>
         private readonly ReadOnlyMemory<byte> _bytes;
 
+        /// <summary>
+        /// Returns an empty BinaryData.
+        /// </summary>
+        public static BinaryData Empty { get; } = new BinaryData(Array.Empty<byte>());
```

Comment:
Nit: I'd use `ROM<byte>.Empty` or `default(ROM<byte>)`. No need to have the static singleton root the empty array.

Reply:
Good catch, thanks for teaching me about the existence of `ReadOnlyMemory<T>.Empty`.  Amended the commit and force pushed.